frontend/src/Admin/components/StudentsTable.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
```

Comment:
This file is copied from the UserTable file with the hometown added (which is a form that can be edited)